rabbitmq.drush.inc

```diff
@@ -92,12 +91,12 @@ function drush_rabbitmq_queue_info($queueName = NULL) {
  */
 function drush_rabbitmq_worker($queueName) {
   /** @var \Drupal\rabbitmq\Consumer $consumer */
-  $consumer = (new Drupal())->service('rabbitmq.consumer');
+  $consumer = \Drupal::service('rabbitmq.consumer');
 
   // Service might be called from a non-Drush environment, so drush_get_option()
   // may not be available to it.
   $consumer->setOptionGetter(function(string $name) {
-    return intval(drush_get_option($name, Consumer::OPTIONS[$name]));
+    return (int) drush_get_option($name, Consumer::OPTIONS[$name]);
```

Comment:
According to PHP Inspections (EA Extended), type casting [can be up to 6 times faster](https://github.com/kalessil/phpinspectionsea/blob/master/src/test/resources/fixtures/api/type-casting-php4-functions.php).
https://3v4l.org/E7soS